README.md

```diff
@@ -124,6 +124,29 @@ If you want to check what resources are going to be targeted without actually te
 cloud-nuke aws --resource-type ec2 --dry-run
 ```
 
+### Ignoring list/nuke errors for certain/all resources
+
+Ideally you should have AWS IAM permissions to list and nuke all target resources. If you do not have permissions to
+nuke resource X - you should exclude it using the `--exclude-resource-type` flag. However, if you don't know which resource
+types you are not permitted to nuke or you have list acceess for all but nuke excess for some, you can ignore their
+errors by specifying the `--ignore-errors` flag:
+
+```shell
+cloud-nuke aws --ignore-errors ec2
+```
+
+This will ignore any errorrs encountered while listing/nuking EC2 resources - please note that if there are resources that
```

Comment:
Please specify what other values you can pass here beyond `ec2` and how to handle multiple values.

Reply:
Also, does this ignore an EC2 error and continue nuking EC2 instances... Or does it stop nuking EC2 instances at the first EC2 error, but then keeps nuking other types of resources? We should make the behavior clear in the docs to avoid surprises.

Reply:
Thanks for the feedback - clarified with example. It keeps on nuking other resource types - updated the README.